stable/datadog/templates/container-system-probe.yaml

```diff
@@ -7,6 +7,7 @@
       add: ["SYS_ADMIN", "SYS_RESOURCE", "SYS_PTRACE", "NET_ADMIN", "IPC_LOCK"]
   command: ["/opt/datadog-agent/embedded/bin/system-probe", "--config=/etc/datadog-agent/system-probe.yaml"]
   env:
+    {{- include "containers-common-env" . | nindent 4 }}
```

Comment:
The system-probe don't need te common environment variable since he doesn't need to communicate the Datadog APIs.

Reply:
Removed the common-env from system probe